gulpfile.babel.ts

```diff
@@ -351,84 +331,107 @@ gulp.task('types', async () => {
+      new webpack.SourceMapDevToolPlugin({
+        fileContext: '../',
+        // Adds hash to the end of the sourcemap URL so that
+        // remote code source maps are not cached erroneously by Sentry
+        // if and when https://github.com/getsentry/sentry/issues/54819 is patched up.
+        filename: '[file].[contenthash].map',
+        publicPath: 'https://www.inboxsdk.com/build/',
+      }),
+    ],
+  };
 } else if (args.integratedPageWorld) {
   // non-remote bundle built for compatibility with remote bundle
-  gulp.task(
-    'sdk',
-    gulp.series('pageWorld', function sdkBundle() {
-      return webpackTask({
-        devtool: 'inline-source-map',
-        entry: './src/inboxsdk-js/inboxsdk-NONREMOTE-INTEGRATED-PAGEWORLD',
-        destName: sdkFilename,
-        standalone: 'InboxSDK',
-        afterBuild: setupExamples,
-        outputLibraryType: OutputLibraryType.Var,
-      });
-    })
-  );
-  gulp.task('remote', () => {
-    throw new Error('No separate remote bundle in non-remote bundle mode');
-  });
-  gulp.task('default', gulp.parallel('sdk', 'types'));
+  config = {
+    ...remoteCompatConfig,
+    devtool: 'inline-source-map',
+    entry: {
+      [sdkFilename]: {
+        library: {
+          export: 'default',
+          name: 'InboxSDK',
+          type: OutputLibraryType.UMD,
+        },
+        import: './src/inboxsdk-js/inboxsdk-NONREMOTE-INTEGRATED-PAGEWORLD',
+      },
+    },
+  };
 } else {
   // standard npm non-remote bundle
-  gulp.task('sdk', async () => {
-    return webpackTask({
-      entry: './src/inboxsdk-js/inboxsdk-NONREMOTE',
-      destName: sdkFilename,
-      disableMinification: true,
-      standalone: 'InboxSDK',
-      afterBuild: async () => {
-        setupExamples();
-        const { minify } = await import('terser');
-
-        const sourceOutput = await fs.promises.readFile(
-          'packages/core/inboxsdk.js',
-          'utf8'
-        );
-
-        const minified = await minify(sourceOutput);
-        await fs.promises.writeFile(
-          'packages/core/inboxsdk.min.js',
-          minified.code!,
-          {
-            encoding: 'utf8',
-          }
-        );
+  config = {
+    entry: {
+      ...pageWorld,
+      [sdkFilename]: {
+        library: {
+          export: 'default',
+          name: 'InboxSDK',
+          type: OutputLibraryType.UMD,
+        },
+        import: './src/inboxsdk-js/inboxsdk-NONREMOTE',
       },
-    });
-  });
-  gulp.task('remote', () => {
-    throw new Error('No separate remote bundle in non-remote bundle mode');
-  });
-  gulp.task('default', gulp.parallel(gulp.series('pageWorld', 'sdk'), 'types'));
+    },
+    disableMinification: true,
+    afterBuild: async () => {
+      setupExamples();
+      const { minify } = await import('terser');
+
+      const sourceOutput = await fs.promises.readFile(
+        'packages/core/inboxsdk.js',
+        'utf8'
+      );
+
+      const minified = await minify(sourceOutput);
```

Comment:
In the interest of reducing build target support, we may remove this output.

We could always bring it back if a user needs it.